napari/layers/vectors/vectors.py

```diff
@@ -679,17 +683,17 @@ def _make_slice_request(self, dims) -> _VectorSliceRequest:
         # things either by caching the world-to-data transform on the layer
         # or by lazily evaluating it in the slice task itself.
         slice_indices = slice_input.data_indices(
-            self._data_to_world.inverse, round_index=False
+            self._data_to_world.inverse, round_index=True
```

Comment:
You're changing `round_index` from `False` to `True` for points and vectors here, but I'm not sure why. At the current commit, I think `round_index` is always `True`, so if you can justify the change, we could just delete it entirely.

Reply:
I think this could be a leftover from refactors... I remember we decussed this line before elsewhere; was the conclusion that points and vectors should both have no rounding?